media/css/firefox/all.less

```diff
@@ -349,11 +350,14 @@ body.nightly {
         margin-left: @gridGutterWidth;
         margin-right: @gridGutterWidth;
         input[type=search] {
-            .span(3);
+            width: 100%;
             float: none;
             margin-left: 0;
             vertical-align: middle;
         }
+        button {
+            width: 100%;
```

Comment:
This could also do with a `margin-top: 10px` to give a little spacing between the input and the button.